open_fdd/desktop/ui/main_window.py

```
@@ -797,7 +799,17 @@ def _run_rules_job(self, site_id: str, source: str, rules_path: str, chunk: int,
         )
         if frame.empty:
             return "No Feather data found for site/source in selected time window."
-        out = run_rule_loop_batched(frame, RuleLoopConfig(rules_path=rules_path, chunk_rows=chunk))
+        model = self.model_service.load()
+        by_model = build_column_map_from_model_points(model, site_id)
+        try:
+            ttl_map = BrickService(ttl_path=self.ttl_service.ttl_path).resolve_column_map()
+        except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)